br/pkg/lightning/importer/table_import.go

```diff
@@ -1034,6 +1037,16 @@ func (tr *TableImporter) postProcess(
 
 			var remoteChecksum *local.RemoteChecksum
 			remoteChecksum, err = DoChecksum(ctx, tr.tableInfo)
+			failpoint.Inject("checksum-error", func() {
+				logutil.CL(ctx).Debug("failpoint checksum-error injected.")
+				err = status.Error(codes.Unknown, "Checksum meets error.")
```

Comment:
`remoteChecksum` should also set to nil. In order to mimic a function returning error, we should handle all the return values. Then you will notice a bug.

Reply:
get it, modified